diff -up LibRaw-0.20.0/libraw.pc.in.1~ LibRaw-0.20.0/libraw.pc.in
--- LibRaw-0.20.0/libraw.pc.in.1~	2020-07-26 01:38:12.224565402 +0200
+++ LibRaw-0.20.0/libraw.pc.in	2020-07-26 01:39:59.908564926 +0200
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: libraw
 Description: Raw image decoder library (non-thread-safe)
-Requires: @PACKAGE_REQUIRES@
+Requires.private: @PACKAGE_REQUIRES@
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lraw -lstdc++@PC_OPENMP@
-Cflags: -I${includedir}/libraw -I${includedir}
+Libs: -lraw@PC_OPENMP@
+Cflags: -I${includedir}/libraw
diff -up LibRaw-0.20.0/libraw_r.pc.in.1~ LibRaw-0.20.0/libraw_r.pc.in
--- LibRaw-0.20.0/libraw_r.pc.in.1~	2020-07-26 01:38:12.224565402 +0200
+++ LibRaw-0.20.0/libraw_r.pc.in	2020-07-26 01:39:46.136437027 +0200
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: libraw
 Description: Raw image decoder library (thread-safe)
-Requires: @PACKAGE_REQUIRES@
+Requires.private: @PACKAGE_REQUIRES@
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lraw_r -lstdc++@PC_OPENMP@
-Cflags: -I${includedir}/libraw -I${includedir}
+Libs: -lraw_r @PC_OPENMP@
+Cflags: -I${includedir}/libraw
